contributors/devel/cri-validation.md

```diff
@@ -34,15 +41,13 @@ This will
 - Run the tests using `ginkgo`
 - Output the test results to STDOUT
 
-critest connects to `/var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set in two ways:
-
-- By setting flags `--runtime-endpoint` and `--image-endpoint`
-- By setting environment variables `CRI_RUNTIME_ENDPOINT` and `CRI_IMAGE_ENDPOINT`
+critest connects to `unit:///var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set by flags `-runtime-endpoint` and `-image-endpoint`.
```

Comment:
s/unit/unix

Is it `-runtime-endpoint` or `--runtime-endpoint`?

Reply:
ACK, those options are Go flags, so `-runtime-endpoint` is right. Also additional flags are in same format.